gum-gum-htb-validator.js

```diff
@@ -43,9 +43,29 @@ var partnerValidator = function (configs) {
                     '*': {
                         type: 'object',
                         properties: {
-                            placementId: {
+                            exec: function (schema, post) {
+                                if (!((post.hasOwnProperty('inScreen') || post.hasOwnProperty('inSlot')))) {
+                                    this.report("Need either inScreen or inSlot");
+                                }
+                                return post;
+                            },
+                            inScreen: {
```

Comment:
Can you please clarify the difference between inScreen slot and inSlot slot? How do you identify them on a client's page?

Reply:
This is dependent on what the publisher wants to request, either an InScreen ad or a classic Slot ad.
They would configure one or the other accordingly. This is pretty much the same setup we offer in our Prebid implementation.